File: src/main/java/com/twoguis/carfixer/model/Agendamento.java
package com.twoguis.carfixer.model;

import java.sql.Date;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Agendamento {
    private int id_agendamento;
    private Date data;
    private int id_tipo_servico;
    private Status status;
    private int id_veiculo;
    private List<Servico> servicos;
}

File: src/app/model/agendamento.ts
export class Agendamento {
    id: number;
    data: string;
    status: string;
    veiculo_id: number;
    tipo_servico_id: number;

    constructor(){
        this.id = 0;
        this.data = "";
        this.status = "";
        this.veiculo_id = 0;
        this.tipo_servico_id = 0;
    }
}
